docs/operation/SOP/PM.OWB.1/PM.OWB.1-ManualPipelineExecution.md

```diff
@@ -0,0 +1,48 @@
+# Manual Pipeline Execution
+
+- Version: 1.0
+- Contact: Alexis Lucattini, [alexisl@unimelb.edu.au](mailto:alexisl@unimelb.edu.au)
+
+Table of Contents
+
+- [Introduction](#introduction)
+- [Requirements](#requirements)
+- [Procedure](#procedure)
+- [Confirmation](#confirmation)
+
+## Introduction
+
+This Pipeline Manager manages the execution of the Oncoanalyser WGTS DNA pipeline. Here we describe the SOP for manual execution of the pipeline.
+
+## Requirements
+
+- appropriate AWS permissions
+- AWS credentials set up in the local environment
+- tools installed
+  - AWS CLI
+  - JQ
+
+## Procedure
+
+To initate a pipeline execution we need to generate an initial DRAFT event. For more details consult the main [README](../../../../README.md).
```

Comment:
Similar to the other SOP, perhaps better to reference the root readme with absolute rather than relative path `(/README.md)